File: javaScript/Leccion9/Empleado.js
class Empleado extends Persona {

    static contadorEmpleados = 0;

    constructor(nombre, apellido, edad, sueldo) {
        super(nombre, apellido, edad);
        this._sueldo = sueldo;
        this.idEmpleado = ++Empleado.contadorEmpleados;
        this.sueldo = sueldo;
    }
    get idEmpleado() {
        return this._idEmpleado;
    }

    get sueldo() {
        return this._sueldo;
    }

    set sueldo(sueldo) {
        this._sueldo = sueldo;
    }

    tostring() {
        return `
        ${super.tostring()} 
        ${this._idEmpleado} 
        ${this._sueldo}`;
    }

}

File: Python/Leccion13/Empleado.py
class Empleado: #Clase Padre
    def __init__(self, nombre, sueldo): #Constructor
        self.nombre = nombre
        self.sueldo = sueldo

    def __str__(self): 
        return f'Empleado: [Nombre: {self.nombre}, Sueldo: {self.sueldo}]'
    
    def mostrar_detalles(self):
        return self.__str__()